### DISCUSSION TITLE
Question about try/catch's in ts-redux-realworld-example-app

### DISCUSSION THREAD

**Discussion by spetca** [realworld-apps/realworld#823] [Q&A]
I have a question about [this file](https://github.com/angelguzmaning/ts-redux-react-realworld-example-app/blob/master/src/services/conduit.ts). Taking updateArticle and getArticle as an example, some of the functions in this file omit try/catch's. Is it to obfuscate back end failure from the front end - i.e. the only errors we care about are errors that were caused by sending data from the front end? I see that the only places that have try/catch's are where axios is receiving a second argument 
`
wondering if anyone knows the reasoning for omitting try catches on some of these? or has any resources to read about this? In short, why wouldnt you use always use a Result type and thus a try/catch scheme, at least for the sake of consistency within this file? 

thanks!

**Comment by geromegrignon** [comment]
Hey @spetca, I encourage you to ask the question by opening an issue in the related repository.
By being a community contribution, this implementation isn't fully reviewed and its maintainer will be the perfect match to get an answer.
